<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1001</width>
    <height>803</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QScrollArea" name="scrollArea">
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>768</width>
        <height>781</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="image_label">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPushButton" name="restore_button">
       <property name="text">
        <string>Restore</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QFormLayout" name="formLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Model:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QComboBox" name="model_combo_box">
         <item>
          <property name="text">
           <string>V1.3</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>V1.2</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Use Real-ESRGAN:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QCheckBox" name="use_real_esrgan_check_box">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Background tile:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QSpinBox" name="background_tile_spin_box">
         <property name="toolTip">
          <string>Tile size for background sampler, 0 for no tile during testing</string>
         </property>
         <property name="minimum">
          <number>0</number>
         </property>
         <property name="maximum">
          <number>10000</number>
         </property>
         <property name="value">
          <number>400</number>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Upscale factor:</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QSpinBox" name="upscale_factor_spin_box">
         <property name="toolTip">
          <string>The final upsampling scale of the image</string>
         </property>
         <property name="minimum">
          <number>1</number>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>Only center face:</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QCheckBox" name="only_center_face_check_box">
         <property name="toolTip">
          <string>Only restore the center face</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QPushButton" name="apply_button">
       <property name="text">
        <string>Apply</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>restore_button</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>restore_face()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>849</x>
     <y>32</y>
    </hint>
    <hint type="destinationlabel">
     <x>782</x>
     <y>8</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>apply_button</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>apply()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>857</x>
     <y>785</y>
    </hint>
    <hint type="destinationlabel">
     <x>785</x>
     <y>761</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>apply()</slot>
  <slot>restore_face()</slot>
 </slots>
</ui>
